_includes/read-time.html

```diff
@@ -0,0 +1,6 @@
+{% assign words = include.content | number_of_words %}
```

Comment:
Do wrap this block of within a css selector so that it's easy to add styles later.

Reply:
Done